bower.json

```diff
@@ -8,5 +8,9 @@
     ".*",
     "HISTORY.md",
     "SpecRunner.html"
-  ]
+  ],
+  "dependencies": {
+    "jasmine": ">=2",
+    "jquery": ">=2"
```

Comment:
I was using [this](https://github.com/velesin/jasmine-jquery#dependencies) part of the docs as reference, where it says that v2 or higher is required.